BlazarUI/app/scripts/models/Log.js

```diff
@@ -14,8 +13,27 @@ class Log extends Model {
   }
 
   url() {
-    return this.logUrl;
+    return `${config.apiRoot}/build/${this.buildNumber}/log/`;
   }
+
+  formatLog(jqxhr) {
+    let cache = [];
+
+    if (jqxhr.status !== 200) {
+      console.warn(jqxhr);
+      return "<p class='x-y-roomy'>Error loading build log. Please check your console for more detail.</p>";
+    }
+
+    const lines = jqxhr.responseJSON.data.split('\n');
+
+    lines.forEach( (line) => {
+      cache.push(`<p class='log-line'>${line}</p>`);
+    });
+
+    return cache.join('\n');
```

Comment:
Yeah I understand the initial splitting by `\n` but I'm not sure why you're doing a `.join('\n')` after creating the `p` elements. You're outputing:

```html
<p class='log-line'>Foo</p>\n
<p class='log-line'>Foo</p>\n
<p class='log-line'>Foo</p>\n
<p class='log-line'>Foo</p>\n
```

Maybe I'm missing something?